frontend/src/scenes/data-warehouse/settings/DataWarehouseManagedSourcesTable.tsx

```diff
@@ -91,35 +58,14 @@ export function DataWarehouseManagedSourcesTable(): JSX.Element {
                     },
                 },
                 {
-                    title: 'Source Type',
+                    title: 'Source',
                     key: 'name',
                     render: function RenderName(_, source) {
-                        return source.source_type
-                    },
-                },
-                {
-                    title: 'Table prefix',
-                    key: 'prefix',
-                    render: function RenderPrefix(_, source) {
-                        return source.prefix
-                    },
-                },
-                {
-                    title: 'Sync Frequency',
-                    key: 'frequency',
-                    render: function RenderFrequency(_, source) {
                         return (
-                            <LemonSelect
```

Comment:
I think the more actions here should now have an "configure" or "view" button incase people don't realize you can click on the source name itself